pkg/providers/instance/types.go

```diff
@@ -46,7 +47,10 @@ func NewInstance(out *ec2.Instance) *Instance {
 		Type:         aws.StringValue(out.InstanceType),
 		Zone:         aws.StringValue(out.Placement.AvailabilityZone),
 		CapacityType: lo.Ternary(out.SpotInstanceRequestId != nil, v1alpha5.CapacityTypeSpot, v1alpha5.CapacityTypeOnDemand),
-		Tags:         lo.SliceToMap(out.Tags, func(t *ec2.Tag) (string, string) { return aws.StringValue(t.Key), aws.StringValue(t.Value) }),
+		SecurityGroupIDs: lo.Map(out.SecurityGroups, func(securitygroup *ec2.GroupIdentifier, _ int) string {
```

Comment:
Can this value also be set for `NewInstanceFromFleet`? It's probably fine if not, but we should keep this consistent if we can